docs/sources/tempo/configuration/_index.md

```diff
@@ -873,6 +873,10 @@ storage:
             # enable to use path-style requests.
             [forcepathstyle: <bool>]
 
+            # optional.
+            # enable to use dualstack endpoint for DNS resolution.
```

Comment:
Could you link to S3 dualstack docs?

https://docs.aws.amazon.com/AmazonS3/latest/userguide/dual-stack-endpoints.html